control-plane/connect-inject/common/common.go

```diff
@@ -73,6 +73,20 @@ func PortValue(pod corev1.Pod, value string) (int32, error) {
 	return int32(raw), err
 }
 
+// WorkloadPortName returns the container port's name if it has one, and if not, constructs a name from the port number
+// and adds a constant prefix. The port name must be 1-15 characters and must have at least 1 alpha character.
+func WorkloadPortName(port *corev1.ContainerPort) string {
+	name := port.Name
+	var isNum bool
+	if _, err := strconv.Atoi(name); err == nil {
```

Comment:
codegolf, nonblocking, unnecessary optimization:

You don't have to try to `Atoi` the empty string, since you know it won't work.

Reply:
Ah right, makes sense, I may skip for now because just trying to get the tests green to merge before code freeze 🤞